src/navigators/types.tsx

```diff
@@ -26,6 +26,8 @@ export type TabParamList = {
   HomeTab: undefined;
   NotificationsTab: undefined;
   SettingsTab: NavigatorScreenParams<SettingsStackParamList>;
+} & {
+  [keyof: string]: undefined;
```

Comment:
I'm wondering if there is another way to do this where we could preserve these types without adding the equivalent of an any here. I'm not totally sure what that would be so I am approving as is.